### PR TITLE
[Community] Added bart-kalanski profile

### LINKED ISSUE #1139: [Community] Recognizing Bart Kalanski as a Contributor and Community Member
**Current Behavior**
@bartkalanski has been a consistent contributor and community member.

**Desired Behavior**
A profile for @bartkalanski should be created and visible on `/community/members` on both the current layer5.io site.

- Github: https://github.com/bartkalanski
- Twitter: bartkodes
- LinkedIn: bart-kalanski
- Profile picture: https://pbs.twimg.com/profile_images/1271353670211375106/n4l3xu3k_400x400.jpg 

---
**Contributor Resources**
The `layer5` repo contains two websites. The current generation and the next-generation of the layer5.io site.

If the `layer5-ng` label is absent on this issue, then this issue pertains to the current generation of the layer5.io website, which uses Jekyll and GitHub Pages. Site content is found under the [`master` branch](https://github.com/layer5io/layer5/tree/master).
- See [`layer5` contributing instructions](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md)

If the `layer5-ng` label is present on this issue, then this issue pertains to the next-generation of the layer5.io website, which uses Gatsby, Strapi, and GitHub Pages. Site content is found under the [`layer5-ng` branch](https://github.com/layer5io/layer5/tree/layer5-ng).
- See [`layer5-ng` contributing instructions](https://github.com/layer5io/layer5/blob/layer5-ng/CONTRIBUTING.md)

### PR DESCRIPTION
**Pull request by Aniruddha-Shriwant**
Signed-off-by: Aniruddha Shriwant <aniruddhashriwant@gmail.com>

**Description**

This PR fixes #1139 

**Notes for Reviewers**

**[Signed commits](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

File: collections/_members/bart-kalanski.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+---
+layout: person
+name: Bart Kalanski
+position: Contributor
+image_path:  https://pbs.twimg.com/profile_images/1271353670211375106/n4l3xu3k_400x400.jpg
```
Comment (Tanuj22) [inline collections/_members/bart-kalanski.md:5]: Can you please download the image and keep it locally like in other members profile?

File: collections/_members/bart-kalanski.md [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+twitter: bartkodes
+github: https://github.com/bartkalanski
```
Comment (Tanuj22) [inline collections/_members/bart-kalanski.md:7]: This will just be `bartkalanski`

**Review by Tanuj22 — CHANGES_REQUESTED** [review]

File: collections/_members/bart-kalanski.md (hunk 1 above)
Comment (Aniruddha-Shriwant) [reply collections/_members/bart-kalanski.md:5]: Ok sure I will commit again

File: collections/_members/bart-kalanski.md (hunk 2 above)
Comment (Aniruddha-Shriwant) [reply collections/_members/bart-kalanski.md:7]: Ohh sorry for that :)
I will commit the changes

**Review by bartkalanski — COMMENTED** [review]
Nicely done!

**Review by bartkalanski — APPROVED** [review]

**Review by Tanuj22 — APPROVED** [review]
Thank You!

**Pull request merged**